### ISSUE TITLE
'Taskbar Segments' option does not show tray icon, system icons or anything

### ISSUE THREAD

**Issue by madhvan-g** [ChrisAnd1998/TaskbarX#942]
![image](https://user-images.githubusercontent.com/58142294/163963422-d9e37963-7b9f-405f-932c-9c04ba00d0a1.png)

**Comment by CengoleFHV** [comment]
yes i just saw the Taskbar Segment Feature and loved it so much but sadly i use my Tray far too often to use that feature without my tray 😢

**Comment by santidellera** [comment]
Came here with the same issue. I hope that in a next update they'd add a feature for that

**Comment by bknie1** [comment]
Just chiming in- I expected Segment to create two bar entities, or at least would allow me to configure them separately. 'Segment' should probably be changed to 'Hide tray area' or something, because that's all it seems to do on mine.
